music_assistant/server/controllers/metadata.py

```diff
@@ -273,35 +272,35 @@ async def get_image_url_for_item(
             for img in media_item.metadata.images:
                 if img.type != img_type:
                     continue
-                if img.source != "http" and not allow_local:
+                if img.provider != "url" and not resolve:
                     continue
-                if img.source != "http" and resolve_local:
-                    # return imageproxy url for local filesystem items
+                if img.provider != "url" and resolve:
+                    # return imageproxy url for images that need to be resolved
                     # the original path is double encoded
                     encoded_url = urllib.parse.quote(urllib.parse.quote(img.url))
                     return f"{self.mass.webserver.base_url}/imageproxy?path={encoded_url}"
                 return img.url
```

Comment:
I think this should be img.path, right?

Reply:
oops, yes and thats why we invented code reviews :-)

Reply:
fixed it on main